autopush/router/__init__.py

```diff
@@ -17,19 +17,20 @@
 from autopush.router.fcm import FCMRouter
 from autopush.settings import AutopushSettings  # noqa
 
-__all__ = ["APNSRouter", "FCMRouter", "GCMRouter", "SimpleRouter",
-           "WebPushRouter"]
+__all__ = ["APNSRouter", "FCMRouter", "GCMRouter", "WebPushRouter"]
```

Comment:
Yeah, I was sorta debating that. We don't have anything that doesn't explicitly import `SimpleRouter`, and since it's deprecated, I figured it would be useful to make sure that nothing is implicitly importing it either. 

I'm fine adding it back in since it technically still exists.